compiler/rustc_passes/src/liveness.rs

```diff
@@ -333,7 +304,7 @@ impl IrMaps<'tcx> {
     }
 
     fn lnk(&self, ln: LiveNode) -> LiveNodeKind {
```

Comment:
imo it's probably better to remove this function and use `self.lnks[ln]` everywhere

Reply:
Done.